sample.env

```diff
@@ -1,8 +1,8 @@
 VALIDATIONTOKEN=token used to validate messenger webhooks
-APPSECRET= appsecret key got on creating facebook app
-PAGEACCESSTOKEN= access token for the page to which the app has subscribed
-FACEBOOKSNS= sns topic name for facebok
-DEFAULT_LANG=en
+APPSECRET=app secret key got on creating facebook app
+PAGEACCESSTOKEN=access token for the page to which the app has subscribed
+FACEBOOKSNS=sns topic name for facebok
+DEFAULT_LANG=in
```

Comment:
for petabencana bot it should be id/ for all others it should be en

Reply:
done (forgot to respond here in the comments)